src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MongoDbSessionHandler.php

```diff
@@ -108,9 +111,16 @@ public function close()
      */
     public function destroy($sessionId)
     {
-        $this->getCollection()->remove(array(
-            $this->options['id_field'] => $sessionId,
-        ));
+        if($this->getCollection() instanceof \MongoDB\Collection)
```

Comment:
Please, respect CS. You should take a look about [PSR-2](http://www.php-fig.org/psr/psr-2/#1-1-example) and [Symfony CS](http://symfony.com/doc/current/contributing/code/standards.html).